src/components/opening-hours-rows/OpeningHoursRows.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { Fragment } from 'react';
```

Comment:
Would opening-hours-preview describe more what this is?

Reply:
And then the other one could be form-opening-hours-preview since it is a wrapper for this